lib/miq_automation_engine/engine/miq_ae_event.rb

```diff
@@ -31,10 +33,13 @@ def self.raise_synthetic_event(event, inputs, message = nil)
       aevent      = build_evm_event(event, inputs)
     end
 
-    MiqAeEngine.deliver(obj_type, obj_id, aevent, instance, nil, nil, message)
+    user = automate_user
+    MiqAeEngine.deliver(obj_type, obj_id, aevent, instance, nil, user.id, message)
```

Comment:
@kbrock 
This code needs to be refactored in a different PR to use MiqAeEngine.resolve_automation_object and that has the ability to pass in the user object.